### ISSUE TITLE
NPE + X11 server crash

### ISSUE THREAD

**Issue by GoogleCodeExporter** [joemasterjohn/debugvisualisation#66]
```
What steps will reproduce the problem?
1. Check out JAXB source code from https://svn.java.net/svn/jaxb~version2
2. Downloaded SMIL 3.0 DTDs from http://www.w3.org/TR/SMIL/DTD.zip
3. Unzip DTD and edit the following lines as explained below:
file smil-attribs-1.mod line 218:
%SMIL.pfx;backgroundOpacity     CDATA    '100' 
file SMIL-control.mod line 73:
bandwidth     CDATA             '100'   
(Make default value 100 instead of 100% otherwise another JAXB bug is triggered)
4. Add a debug breakpoint on the first line of:
com.sun.tools.xjc.reader.ModelChecker [line: 113] - 
checkPropertyCollision(CPropertyInfo, CPropertyInfo)
5. Create the following java application launch configuration:
main: com.sun.tools.xjc.XJCFacade
args: -dtd -d /output/dir  -p org.w3c.smil30 /path/to/DTD/SMIL30Language.dtd
6. Debug XJC using the above launch config.
7. Upon hitting the breakpoint from step 4, visualize
this>model>beans content in the Debug Visualization view.

What is the expected output? What do you see instead?
At some point (e.g. double click on 'beans') the whole x server crashed on me. 
I'm on arch linux 64 bit. Because the X server crashed I was unable at first to 
get any log info. I had to start eclipse from a console running outside of X 
(Ctrl-Alt-F1 , $>export DISPLAY=:0.0 , $>eclipse.sh , Ctrl-Alt-F7) to have the 
log info, see attached file.

In short it looks like a java.lang.NullPointerException
    at hu.cubussapiens.debugvisualisation.views.handlers.AbstractGraphCommandHandler.getGraphViewer(AbstractGraphCommandHandler.java:56)

What version of the product are you using? On what operating system?
Eclipse 3.6.1 64bit on Arch linux 64bit, KDE 4.5.2.

Please provide any additional information below.

```

Original issue reported on code.google.com by `fmjrey` on 4 Jan 2011 at 12:01

Attachments:
- [debugvisualisation.issue66.txt](https://storage.googleapis.com/google-code-attachments/debugvisualisation/issue-66/comment-0/debugvisualisation.issue66.txt)
